index.html

```diff
@@ -537,6 +542,12 @@
         browsing context and can possibly disclose sensitive information about
         the state of the device such as that battery level is low.
       </p>
+      <p>
+        A <dfn>wake lock</dfn> can only be granted when requested within a
+        <a href="https://w3c.github.io/webappsec/specs/powerfulfeatures/#secure-context">secure context</a>. This means that any wake lock request should come from
```

Comment:
Also, please remove "a page served over HTTPS". Secure contexts are not just HTTP, they can include a bunch of other protocols - including `http://localhost`.